<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Caslon Pro&quot;;
font: 20pt &quot;华文中宋&quot;;</string>
   </property>
   <property name="text">
    <string>               欢迎登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountlineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordlineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认登陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeProgram">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
